lms/static/scripts/frontend_apps/components/test/SubmitGradeForm-test.js

```diff
@@ -76,6 +92,33 @@ describe('SubmitGradeForm', () => {
     assert.equal(wrapper.find('input').prop('defaultValue'), '');
   });
 
+  [
+    {
+      text: 'focuses the input field when changing students',
```

Comment:
I don't know that this is an accurate test name if the hook is triggered by changes to `grade`.

Reply:
I'll reword the test to include a bit about fetching the grade, but the test I want to run is changing a student. Subsequently, each students grade is auto-fetched. The only prop control we have is changing a student, not forcing a new grade to be re-fetched.